crates/topos-sequencer-subnet-runtime/src/lib.rs

```diff
@@ -18,6 +18,8 @@ pub mod proxy;
 
 use crate::proxy::{SubnetRuntimeProxyCommand, SubnetRuntimeProxyEvent};
 
+const EVENT_SUBSCRIBER_CHANNEL_SIZE: usize = 64;
```

Comment:
We should have a comment here about the reasoning for chosing `64` and not some other number; even if it's just "off the top of my head".